fuzz/ci-fuzz.sh

```diff
@@ -13,10 +13,17 @@ rm *_target.rs
 [ "$(git diff)" != "" ] && exit 1
 popd
 
+export RUSTFLAGS="--cfg=secp256k1_fuzz --cfg=hashes_fuzz"
+
+mkdir -p hfuzz_workspace/full_stack_target/input
+pushd write-seeds
+RUSTFLAGS="$RUSTFLAGS --cfg=fuzzing" cargo run ../hfuzz_workspace/full_stack_target/input
```

Comment:
I wanted to verify that using the hardcoded seed indeed increases coverage, but it seems that hongfuzz reports in the log always

`branch_coverage_percent:0`

Not sure why that is? Otherwise it would be easy to compare with and without this hardcoded seed.

Or maybe there is another way to verify that it works indeed?

Reply:
Okay, indirect evidence. But why is hongfuzz reporting 0?

Reply:
Is it not using coverage to direct fuzzing?

Reply:
Does it show you `branch_coverage_percent` as non-zero locally?